src/libutil/experimental-features.hh

```diff
@@ -93,4 +93,10 @@ public:
 void to_json(nlohmann::json &, const ExperimentalFeature &);
 void from_json(const nlohmann::json &, ExperimentalFeature &);
 
+/**
+ * It is always rendered as a string
+ */
+template<>
+struct json_avoids_null<ExperimentalFeature> : std::true_type {};
```

Comment:
This is about about JSON `null`, not C++ `null`. If `T` already is serialized with `null`, we cannot use `null` for `std::nullopt` because that would be ambiguous. We can only use `null` for that when it isn't already in use.